auto_submit/lib/validations/ci_successful.dart

```diff
@@ -127,13 +133,27 @@ class CiSuccessful extends Validation {
     return allSuccess;
   }
 
+  List<String> _contextNodeListToStringList(List<ContextNode> contextNodeList) {
```

Comment:
Instead of adding this one off function here, we should implement toString/toJson in ContextNode

Reply:
Yeah I though about that. I just wanted to get some logging in now before everyone is gone next week so I can at least see what is going on myself.

Reply:
I'll be here next week, feel free to send PRs my way